pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs

```diff
@@ -313,7 +371,7 @@ private void ComboChanged(object sender, EventArgs e)  // CONSIDER(bspratt) no c
 
             // Grey out any ignored column
             SetColumnColor(comboBox);
-
+            
```

Comment:
In general, it's good practice to avoid non-functional changes to the code, it slows down reviewers